scripts/Synapse/updateECRDataStore.ipynb

```diff
@@ -54,6 +54,10 @@
     "    \"incident_id_date_added\": [\"timestamp\", True],\n",
     "    \"last_name\": [\"string\", True],\n",
     "    \"first_name\": [\"string\", True],\n",
+    "    \"birth_date\": [\"date\", True],\n",
+    "    \"gender_identity\": [\"string\", True],\n",
```

Comment:
For my own clarification, which of the gender vars in the ecr messages is this coming from?

Reply:
Good question! If you look on line 28 of the [Patient resource template](https://github.com/CDCgov/phdi/blob/main/containers/fhir-converter/Templates/eCR/Resource/_Patient.liquid#L28) that we use for eICR conversion you can see that we pull the value from `patientRole.patient.administrativeGenderCode.code`. [Line 40 in this dummy eICR](https://github.com/CDCgov/phdi/blob/main/tests/assets/fhir-converter/ecr/example_eicr_with_rr_data.xml#L40C23-L40C23) is an example of precisely where this data is located in the XML of a raw eICR prior to FHIR conversion.

Reply:
For additional context, [this PR](https://github.com/CDCgov/phdi/pull/691) on the `phdi` repo adds these fields to the parsing schema we are using in the message parser. This is will allow the JSON blobs that the pipeline writes to storage to contain this data so that when the code in this PR runs the data can be parsed appropriately and added to the eCR datastore.